docs/tutorials/remote_client.md

```diff
@@ -60,7 +60,7 @@ sudo systemctl enable --now -s sshd
 #### Setting up SSH
 Remote Podman uses SSH to communicate between the client and server. The remote client works considerably smoother using SSH keys. To set up your ssh connection, you need to generate an ssh key pair from your client machine.
 ```
-ssh-keygen
+ssh-keygen -t ed25519  -f $HOME/.ssh/id_rsa
```

Comment:
The file should not be called is_rsa when it is a ed25519 key. Just use the default name.

Reply:
Make sense, will update it.